pyro/infer/importance.py

```diff
@@ -15,7 +15,59 @@
 from .util import plate_log_prob_sum
 
 
-class Importance(TracePosterior):
+class WeightAnalytics:
```

Comment:
WDYT of making this a little clearer:
- Maybe renaming to `WeightAnalyticsMixin` or `LogWeightsMixin` or something?
- Adding a simple docstring like """Mixin class to compute analytics from a `.log_weights` attribute."""
- Adding just type declaration (but no initialization):
  ```py
  class WeightAnalytics:
      log_weights: Union[List[float], torch.Tensor]
      ...
  ```

Reply:
Thank you for your comments @fritzo! I completely agree with them and have integrated them.